roles/mesos/defaults/main.yml

```diff
@@ -8,7 +8,8 @@ mesos_agent_package: "mesos-agent-{{ mesos_version }}"
 mesos_mode: follower
 mesos_log_dir: /var/log/mesos
 mesos_logging_level: WARNING
-mesos_work_dir: /var/lib/mesos
+mesos_work_dir: /mnt/mesos
+mesos_lvm_data_volume_size: 20%FREE
```

Comment:
not sure if this is a sensible default. on a 1 tb attached disk, the layout would be the following for a mesos worker:

docker: 400gb
mesos: 120gb
mantldata: 480gb